javascript-timers/interval.js

```diff
@@ -0,0 +1,14 @@
+const intID = window.setInterval(function () {
+  const header = document.querySelector('h1');
+  if (header.textContent === '4') {
+    header.textContent = '3';
+  } else if (header.textContent === '3') {
+    header.textContent = '2';
+  } else if (header.textContent === '2') {
+    header.textContent = '1';
+  } else if (header.textContent === '1') {
+    header.textContent = '~Earth Beeeelooowww Us~';
+  } else if (header.textContent === '~Earth Beeeelooowww Us~') {
+    window.clearInterval(intID);
+  }
```

Comment:
There's probably a less brute-force way to do this.

Reply:
![setInt3](https://user-images.githubusercontent.com/75236087/123368928-91bb0a00-d531-11eb-8347-b24e87f4c208.gif)